content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst

```diff
@@ -0,0 +1,240 @@
+      :align: center
+      :alt: Enter 14-digit GTIN into the Barcode field on product form.
+
+.. tip::
+   To view a list of *all* products and their corresponding barcodes in the Odoo database, navigate
+   to :menuselection:`Inventory app --> Configuration --> Settings`. Under the :guilabel:`Barcode`
+   heading, click on the :guilabel:`Configure Product Barcodes` button under the :guilabel:`Barcode
+   Scanner` section. Enter the 14-digit GTIN into the :guilabel:`Barcode` column, then click
+   :guilabel:`Save`.
+
+   .. image:: gs1_usage/product-barcodes-page.png
+      :align: center
+      :alt: View the Product Barcodes page from inventory settings.
+
+.. _barcode/operations/lot-setup-on-product:
+
+After activating lots and serial number tracking in the settings page, specify that this feature is
+to be applied on each product by navigating to the :guilabel:`Inventory` tab on the product form.
+Under :guilabel:`Tracking`, choose the :guilabel:`By Lots` radio button.
+
+.. image:: gs1_usage/track-by-lots.png
+   :align: center
+   :alt: Enable product tracking by lots in the "Inventory" tab of the product form.
+
+Scan barcode on receipt
+-----------------------
+
+To ensure accurate lot interpretation in Odoo on product barcodes scanned during a receipt
+operation, navigate to the :menuselection:`Barcode` app to manage the :ref:`receipt picking process
+<barcode/operations/scan-received-products>`.
+
+From the :guilabel:`Barcode Scanning` dashboard, click the :guilabel:`Operations` button, then the
+:guilabel:`Receipts` button to view the list of vendor receptions to process. Receipts generated
+from :abbr:`POs (Purchase Orders)` are listed, but new receipt operations can also be created
+directly through the :menuselection:`Barcode` app using the :guilabel:`Create` button.
+
+On the list of receipts, click on the warehouse operation (`WH/IN`) and scan product barcodes and
+lot numbers with a barcode scanner. The scanned product then appears on the list. Use the
+:guilabel:`✏️ (pencil)` button to open a window and manually enter quantities for specific lot
+numbers.
+
+.. example::
+   After placing a :abbr:`PO (Purchase Order)` for fifty apples, navigate to the associated receipt
+   in the *Barcode* app.
+
+   Scan the barcode containing the |GTIN|, quantity, and lot number. For testing with a barcode
+   scanner, below is an example barcode for the fifty Fuji apples in Lot 2.
+
+   .. list-table::
+      :header-rows: 1
+      :stub-columns: 1
+
+      * -
+        - 2D Matrix
+        - |AI|
+          (product)
+        - GS1 Barcode (product)
+        - |AI|
+          (quantity)
+        - GS1 Barcode (quantity)
+        - |AI| (lot)
+        - GS1 Barcode (lot #)
+        - Full GS1 barcode
+      * - 50 Fuji apples in Lot0002
+        - .. image:: gs1_usage/fuji-apples-barcode.png
+             :align: center
+             :alt: 2D matrix of GS1 barcode of 50 fuji apples with an assigned lot number.
```

Comment:
@samueljlieber Do you have any suggestions about fixing the image width/heights or scaling the barcode? Even at full size, the barcode image is a little small and I'm wondering if it'll be an issue when the window needs to be minimized/resized.

Reply:
I agree the QR code and table headers do appear to be scrunched in this layout. I'd suggest flipping the table axis so there are two columns, giving more room to breathe and accommodate different screen sizes. I added RST here: https://github.com/odoo/documentation/pull/5592/files/73cd08835811c5e774d4ac0e58fad9e38ac48cb2#r1305710601

<img width="942" alt="Screenshot 2023-08-25 at 10 06 49 AM" src="https://github.com/odoo/documentation/assets/36018073/1ab2259d-ff69-4401-983a-41f14a8cb33b">